src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import { px } from '../../utils';
+
+import Select from '../Select/Select';
+import Button from '../Button/Button';
+import chevronRight from '../../assets/chevronRight.svg';
+
+export interface PaginationProps {
+  /**
+   * Unique id for component testing
+   */
+  id: string;
+  /**
+   * Base class for Pagination component.
+   */
+  className?: string;
+  /**
+   * Child element pass in to display as options to select from.
+   */
+  children?: React.ReactNode;
```

Comment:
I'd rather have this be `pageValues: number[]`, and wrap those in the `<option` before passing as `Select` children